scripts/lst-sensitivity.py

```diff
@@ -21,20 +22,28 @@
 
 args = parser.parse_args()
 
-nfiles_gammas = 100*0.2 #Pointlike gammas
-nfiles_protons = 5000*0.2
+nfiles_gammas = 100 #Pointlike gammas
+nfiles_protons = 5000*0.8
 
-eb = 12 # Number of energy bins
-gb = 10 #Number of gammaness bins
+eb = 20 # Number of energy bins
+gb = 11 #Number of gammaness bins
 tb = 10 #Number of theta2 bins
-obstime = 50 * 3600 *u.s
+obstime = 50 * 3600 * u.s
 noff = 5
 
-E, best_sens = sensitivity.sens(args.simtelfile_gammas, args.simtelfile_protons,
+E, best_sens, result, units, rate_g, rate_p, w_g, w_p, nex_5sigma, final_hadrons = sensitivity.sens(args.simtelfile_gammas, args.simtelfile_protons,
```

Comment:
this function has now maaaaany outputs, it would be nice to break it down

Reply:
Ah yes, sorry, I was doing tests and needed all of those and forgot to remove them before merging. The only ones needed are the first 4 actually.